pybind11.BUILD

```diff
@@ -21,6 +21,7 @@ OPTIONS = select({
 INCLUDES = [
     "include/pybind11/*.h",
     "include/pybind11/detail/*.h",
+    "include/pybind11/eigen/*.h",
```

Comment:
Could you please double-check that this works for you (and update this PR if it does)?

```
INCLUDES = [
    "include/pybind11/**/*.h",
]
```

I tested this locally already by running the pybind11_abseil tests.

I tried this for two reasons:

1. There is also pybind11/stl/*.h
2. What if pybind11/eigen does not exist? (it was released only very recently)

To 1: The recursive glob (`**`) also covers the `stl` subdirectory and is future proof.
To 2: No problem! But the recursive glob sidesteps the question, which seems nicer.